PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Please ensure your pull request adheres to the following guidelines: 
+
+- [ ] Use the following formatting style: [SonarSource/sonar-developer-toolset](https://github.com/SonarSource/sonar-developer-toolset#code-style)
+- [ ] Unit tests are passing and you provided a unit test for your fix
```

Comment:
PR could be an improvement.

This could be phrased: provide a UT for any code you changed